openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -244,9 +244,9 @@ jQuery(function () {
             .then((module) => module.init());
     }
 
-    if (window.READINGLOG_STATS_CONFIG) {
+    if ($('div.readinglog-charts').length) {
```

Comment:
```suggestion
    const readingLogCharts = document.querySelector('.readinglog-charts')
    if (readingLogCharts) {
        const readingLogConfig = JSON.parse(readingLogCharts.dataset.config)
```
Avoid adding new code that relies on jQuery.  Avoid multiple queries for the same element.